pipeline/transformer/transformer_mets/src/main/scala/uk/ac/wellcome/platform/transformer/mets/transformer/Mets.scala

```diff
@@ -59,4 +62,14 @@ case class Mets(
       ontologyType = "Work",
       value = recordIdentifier)
   }
+
+  private val thumbnailHeight = "200"
+
+  private def thumbnail =
+    thumbnailLocation.map { location =>
+      DigitalLocation(
+        s"https://dlcs.io/iiif-img/wellcome/5/$location/full/,$thumbnailHeight/0/default.jpg",
```

Comment:
I thought we said we were going to use the boxed syntax?
ie `https://dlcs.io/iiif-img/wellcome/5/$location/full/!$thumbnailWeight,$thumbnailHeight/0/default.jpg`